packages/api/src/utils/tags.js

```diff
@@ -1,10 +1,22 @@
+/**
+ * @param {import('@web3-storage/db/db-client-types').UserOutput} user
+ * @param {string} tagName
+ * @param {string|undefined} defaultValue
```

Comment:
```suggestion
 * @param {string} [defaultValue]
```
While setting the type to `string|undefined` works I think this is the jsdoc way